aspnetcore/host-and-deploy/iis/hosting-bundle.md

```diff
@@ -20,13 +20,20 @@ The .NET Core Hosting bundle is an installer for the .NET Core Runtime and the [
 >
 > If the Hosting Bundle is installed after installing the 64-bit (x64) version of .NET Core, SDKs might appear to be missing ([No .NET Core SDKs were detected](xref:test/troubleshoot#no-net-core-sdks-were-detected)). To resolve the problem, see <xref:test/troubleshoot#missing-sdk-after-installing-the-net-core-hosting-bundle>.
 
-### Direct download (current version)
+## Direct download (current version)
 
 Download the installer using the following link:
 
 [Current .NET Core Hosting Bundle installer (direct download)](https://dotnet.microsoft.com/permalink/dotnetcore-current-windows-runtime-bundle-installer)
 
-### Earlier versions of the installer
+## Visual C++ Redistributable Requirement
+
+On older versions of Windows, for example Windows Server 2012 R2, you'll need to install the Visual Studio C++ 2015, 2017, 2019 Redistributable. Otherwise, a confusing error messaging saying: `The data is the error.` will be output in the Windows Event Log.
```

Comment:
Drops future tense ...

```suggestion
On older versions of Windows, for example Windows Server 2012 R2, install the Visual Studio C++ 2015, 2017, 2019 Redistributable. Otherwise, a confusing error message in the Windows Event Log reports that `The data is the error.`
```

... might need to code-fence **Visual Studio C++ 2015, 2017, 2019 Redistributable**, too, because MT might over-translate and make that into something mangled. Same thing for the link text below.